tests/Time/PeriodOverlapTest.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+
+namespace Sundata\Utilities\Test\Time;
+
+use Carbon\CarbonImmutable;
+use Sundata\Utilities\Time\Period;
+use PHPUnit\Framework\TestCase;
+
+class PeriodOverlapTest extends TestCase
+{
+
+    public function testPeriodsThatDontOverlapReturnNull()
```

Comment:
dit is een goeie plek om dataproviders te gebruiken lijkt me

Reply:
ja ik vond het juist wel lekker om zo gewoon die paar cases in die file te hebben. Jij niet?